### PR TITLE
Multi gpu followup

### PR DESCRIPTION
**Pull request by abergeron**
Some interface changes to allow for easier use of the multi-gpu and some small bug fixes for unusual circumstances.

### CODE REVIEW

**Comment by nouiz** [comment]
travis is failing.

File: doc/library/tensor/basic.txt [hunk 1]
```diff
@@ -427,7 +427,8 @@ TensorVariable
     you'll want to call.
 
 
-.. class:: _tensor_py_operators(object)
+.. autoclass:: _tensor_py_operators
+   :members:
```
Comment (nouiz) [inline doc/library/tensor/basic.txt]: Why this change? I don't see a change in the doc generated. I don't have objection to this change.

Comment (abergeron) [inline doc/library/tensor/basic.txt]: This should add the documentation of transfer() which is only in its docstring.  We can move the rest later.

Comment (nouiz) [inline doc/library/tensor/basic.txt]: ok

**Comment by abergeron** [comment]
I've restarted the build because it seems to have failed due to a problem with the installation of six.

**Comment by nouiz** [comment]
I have started the tests on my machine for sandbox/gpuarray. If they pass, I'll merge.
